components/admin_console/group_settings/group_details/group_details.test.jsx

```diff
@@ -142,4 +142,39 @@ describe('components/admin_console/group_settings/group_details/GroupDetails', (
         expect(actions.link).toBeCalledWith('xxxxxxxxxxxxxxxxxxxxxxxxxx', '22222222222222222222222222', 'team', {auto_add: true});
         expect(actions.link).toBeCalledTimes(2);
     });
+
+    test('update name for null slug', async () => {
+        const wrapper = shallow(
+            <GroupDetails
+                {...defaultProps}
+                group={{display_name: 'test group', allow_reference: false}}
```

Comment:
I'm not sure what the server-side validations are for the name field. Does `.toLowerCase().replace(/\s/g, '-')` ensure a valid name?

Reply:
No, the display_name could have what we consider "invalid" characters. But that would be verified server side on save, same as duplicate check. We don't necessarily guarantee this name is valid and unique. But is just a default name. If we expect illegal characters (like space) we can replace them. 

Handling this client side brings additional questions, do we replace the illegal characters or warn the user? Do we disable save until valid? Shouldn't we then validate whenever the user makes a change to be complete. I would suggest leaving that check on the server.